core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorSidebarDirective.js

```diff
@@ -36,7 +39,7 @@ oppia.directive('simpleEditorSidebar', [
             'Multiple choice', 'Correct answer', 'Hints', 'Bridge text'];
           $scope.questionList = SimpleEditorManagerService.getQuestionList();
           $scope.ID_PREFIX = QuestionIdService.SIDEBAR_PREFIX;
-
+          $scope.modeSrv = SimpleEditorSidebarModeService;
```

Comment:
I changed from above because @kevinlee12 suggested [here](https://github.com/oppia/oppia/pull/3547#discussion_r122605469) using a smaller name. Above is much clear but it is long and creates long lines when used inside HTML. do you have any other smaller name suggestion or should we revert back to above?